live-examples/js-examples/sharedarraybuffer-slice.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js" class="language-js">// create a SharedArrayBuffer with a size in bytes
+var buffer = new SharedArrayBuffer(16);
+var int32View = new Int32Array(buffer); // create the view
+// produces Int32Array [0, 0, 0, 0]
+
+int32View[1] = 42;
+var sliced = new Int32Array(buffer.slice(4,12));
```

Comment:
Nit: consider a comment for the arguments:

`// slice from index 4 to index 12`